### PR TITLE
collection, our story, footer fixes

### PR DESCRIPTION
**Pull request by teddymaulana**

### CODE REVIEW

File: src/scss/sections/_footer.scss [hunk 1]
```diff
@@ -1,65 +1,85 @@
...
-  }
+	&__support {
+		grid-area: support;
+	}
+
+	.dropdown .dropdown-menu {
+		-webkit-transition: all 0.3s;
+		-moz-transition: all 0.3s;
+		-ms-transition: all 0.3s;
+		-o-transition: all 0.3s;
```
Comment (foria) [inline src/scss/sections/_footer.scss:70]: we don't need these

File: src/scss/sections/_howto.scss [hunk 2]
```diff
@@ -24,34 +24,36 @@
...
-			margin-top: -7em;
-			margin-left: -2em;
+			top: 4.5em;
+			right: 1em;
 
 			@include media-breakpoint-up(lg) {
-				margin-left: -7em;
-				margin-top: -4.5em;
+				right: 6em;
+				top: 12em;
```
Comment (foria) [inline src/scss/sections/_howto.scss:45]: this solution is not responsive

File: src/scss/templates/_our-story.scss [hunk 3]
```diff
@@ -21,6 +22,10 @@
 	}
 }
 
+.page-award-slider {
```
Comment (foria) [inline src/scss/templates/_our-story.scss:25]: duplicate

File: src/scss/components/article-proggress-bar.scss [hunk 4]
```diff
@@ -9,4 +9,9 @@
 		background-color: $secondary;
 		height: 100%;
 	}
+
+	&--sticky {
```
Comment (foria) [inline src/scss/components/article-proggress-bar.scss:13]: I don't see the bar on the article

**Review by foria — CHANGES_REQUESTED** [review]

File: src/scss/sections/_howto.scss (hunk 2 above)
Comment (teddymaulana) [reply src/scss/sections/_howto.scss]: I didn't find the solution, it seems we need to detect the image height to keep the arrow in the center of the images, can't find a way with css, unless we use JS
I am using extra media queries to adjust the arrow position for this solution for now

**Review by foria — APPROVED** [review]

**Pull request merged**